src/api/spec/models/comment_spec.rb

```diff
@@ -13,7 +13,7 @@
   describe 'save' do
     it 'stores emoji' do
       comment_package.body = '😁'
-      comment_package.save
+      expect { comment_package.save }.not_to raise_error
```

Comment:
`save` will return `true` or `false`, so no exception will be raised if the validations don't pass, isn't it?

Reply:
Oh! I missed the bang. This was my intension:
 
```suggestion
      expect { comment_package.save! }.not_to raise_error
```